app/Models/Litter/Categories/Alcohol.php

```diff
@@ -27,24 +27,21 @@ public function photo () {
     	return $this->belongsTo('App\Models\Photo');
     }
 
-    /**
-     * Pre-defined litter types available on this class
-     */
-    public function types ()
+    public function typesForExport(): array
     {
         return [
-            'beerCan',
-            'beerBottle',
-            'spiritBottle',
-            'wineBottle',
-            'brokenGlass',
-            'bottleTops',
-            'paperCardAlcoholPackaging',
-            'pint',
-            'plasticAlcoholPackaging',
-            'six_pack_rings',
-            'alcohol_plastic_cups',
-            'alcoholOther'
+            'beerCan' => 'beer_can',
+            'beerBottle' => 'beer_bottle',
+            'spiritBottle' => 'spirit_bottle',
+            'wineBottle' => 'wine_bottle',
+            'brokenGlass' => 'broken_glass',
+            'bottleTops' => 'beer_bottle_top',
+            'paperCardAlcoholPackaging' => 'paper_card_alcohol_packaging',
+            'pint' => 'pint_glass',
+            'plasticAlcoholPackaging' => 'plastic_alcohol_packaging',
+            'six_pack_rings' => 'six_pack_ring',
+            'alcohol_plastic_cups' => 'alcohol_plastic_cup',
+            'alcoholOther' => 'alcohol_other',
```

Comment:
ideally we should only have 1 key for everything which would make it more dynamic

Reply:
We now have automated tests that prove that the types() array is identical to the db table columns, so there is no risk there at all.
We can totally remove these other keys, but they're not duplicates, they're used in data exports. We can directly use tags instead of these adaptations in exports. Actually, what I think makes more sense, is to completely remove these tag names, and on the CSVExporter use translations instead.
But for the time being we need them. Putting them here makes it easier to test their correctness, and remove the cluttering from csv exporter.

Reply:
Translations would be good, but isn't the types array enough to generate the csv export? This seems like a lot of duplication... it would be cool if we only have to declare each key once

Reply:
We can just go back to the old types() method, but would you be ok with the csv column names to be identical to the db column names? After all, you added the names on the csv to be more expressive, right? When we add translations later to the csv we'll remove this duplication entirely, that's for sure. But with these changes we'll be able to write tests for that.

Reply:
I'm happy for them to be the same as the table column names for now. The problem with adding more code like this is it makes adding new items more difficult. I don't think we will be adding these translations anytime soon. If anyone wants them in their language they will have to open a PR, or see our js translations to match key => value for now

Reply:
Ok cool, I'm fixing it then.

Reply:
@xlcrr I refactored the types array. I also replaced fillable with the $guarded property, on the LitterCategory abstract class, so all the others inherit it, I wrote a test that it works for all categories.
I also deleted the photos() relationship on some of the models that had it. The proper type of relationship depending on the DB foreign keys should have been Smoking->hasOne(Photo), but even that was not working. It wasn't working in any of them, and it's not used anywhere in code.